anchore_manager/cli/service.py

```diff
@@ -340,6 +340,7 @@ def start(services, auto_upgrade, anchore_module, skip_config_validate, skip_db_
                             logger.warn(
                                 "this version of anchore-engine requires the anchore DB version ({}) but we discovered anchore DB version ({}) in the running DB - it is safe to run the upgrade while seeing this message - will retry for {} more seconds.".format(
                                     str(code_versions['db_version']), str(db_versions['db_version']), str(max_timeout - int(time.time() - timer))))
+                            logger.warn("See: https://docs.anchore.com/current/docs/engine/engine_installation/upgrade/#advanced--manual-upgrade-procedure")
```

Comment:
Please remove this message as this is an expected state while the components wait for the system to upgrade itself. The other message you've added handles the case where the retry count completed with no upgrade found and thus the user *may* need to take action.

Reply:
@zhill OK I removed it